src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -2503,22 +2513,28 @@ mongoc_uri_get_option_as_int32 (const mongoc_uri_t *uri,
 
 static void
 _mongoc_uri_warn_for_bad_int_option_combos (const mongoc_uri_t *uri,
-					    const char *option)
+                                            const char *option)
 {
    /* Warn for deprecated option combinations */
    if (!strcasecmp (option, MONGOC_URI_TIMEOUTMS)) {
-      if (mongoc_uri_get_option_as_int64 (uri, MONGOC_URI_WAITQUEUETIMEOUTMS, -1) > 0 ||
-	  mongoc_uri_get_option_as_int64 (uri, MONGOC_URI_SOCKETTIMEOUTMS, -1) > 0 ||
-	  mongoc_uri_get_option_as_int64 (uri, MONGOC_URI_WTIMEOUTMS, -1) > 0) {
-	 MONGOC_WARNING ("Setting a deprecated timeout option %s in combination with timeoutMS", option);
+      if (mongoc_uri_get_option_as_int64 (
```

Comment:
This is due to clang-format, there were no code changes in `_mongoc_uri_warn_for_bad_int_option_combos`.